common/redshift/python2_requirements.txt

```diff
@@ -1 +1,2 @@
-pytest==4.6.11
\ No newline at end of file
+pytest==4.6.11
+numpy==1.8.2
```

Comment:
so are we going to add every module dependencies here?
I think I didn't express well the other possibility. It would be having python2_requirements.txt and python3_requirements.txt also inside every module and module specific libraries would go inside.
For me this is good for now but I would go for the other to avoid all the conflicts that we will have merging.

Reply:
I have this part already implemented in: https://github.com/CartoDB/carto-spatial-extension/compare/add-quadkey-module-to-redshift?expand=1

Also the installer script would install `python2_requirements.txt`. Although maybe we can keep in the common those modules that are already installed in redshift by default... as numpy. So we are not uploading them again. So then you will not have to change anything.

Reply:
I agree with your approach but let's wait @Jesus89 opinion so we can set a standard

Reply:
Let's talk tomorrow about this in the daily

Reply:
We'll include a requirements.txt file per module